sway/commands.c

```diff
@@ -35,6 +35,8 @@
 #include "input.h"
 #include "border.h"
 
+#define ARRAY_SIZE(array) (sizeof(array) / sizeof(array[0]))
```

Comment:
I know you are not a fan of macros but I was hoping you would make an exception here on the basis that this macro improves code quality, is one of the most commonly used macros and that it cannot be implemented as a function. I'm not going to argue with religion though so if you still want me to get rid of it just give it another thumbs down.